Clarify and improve SEO for negation in search

nielsbom opened the pull request:
The Search page did not use the term "negate" or "negation" which is a term people will use to find this feature.

I also added an example of negating an OR expression in parentheses.

marcusolsson commented on en/Plugins/Search.md:
```diff
@@ -31,6 +31,10 @@ To exclude a word from the search results, add a hyphen (`-`) in front of it:
 
+The technical term for excluding words is to "negate" or "negation". You can negate any expression or any group of expressions using parentheses.
+
+- `meeting -(work OR meetup)` returns files that contain `meeting` but do not contain `work` or `meetup`.
```
```suggestion
- `meeting -(work OR meetup)` returns files that contain `meeting` but not `work` or `meetup`.
```

Keep symmetry with previous examples.

marcusolsson reviewed (commented):
Thank you for the suggestions! 🙏 I added a couple of comments. Let me know what you think.

nielsbom commented:
Hi @marcusolsson thanks for the quick response.

Using your feedback I updated my PR:

1. "technical term" yep, let's remove that.
2. Simpler = better.
3. Search engines will find "negate" when searching for "negation" so SEO-wise this will have the same result.


My original example of `meeting -(work OR meetup)` is functionally similar to `meeting -work -meetup` which I think is simpler so I added an example using the latter query.

I also added an example for negation of the logical AND (which uses the ` ` operator 😃).

marcusolsson commented on en/Plugins/Search.md:
```diff
@@ -27,10 +27,18 @@ You can use parentheses to control the priority of each expression.
 - `meeting (work OR meetup) personal` returns files that contain `meeting`, `personal`, and either `work` or `meetup`.
 
-To exclude a word from the search results, add a hyphen (`-`) in front of it:
+To exclude or _negate_ a word from the search results, add a hyphen (`-`) in front of it:
```
```suggestion
To exclude, or _negate_, a word from the search results, add a hyphen (`-`) in front of it:
```

marcusolsson reviewed (changes requested):
Looking over the Search docs, I notice that we don't do a good job at explaining what a query, term, and expression are, and how they relate.

While this PR highlight this problem, I think it can be addressed in a separate PR. Aside from my nitpick, this is good to merge.

nielsbom commented:
🎉
